misc/pkcsslotd.in

```diff
@@ -12,7 +12,7 @@
 
 . /etc/init.d/functions
 
-PIDFILE=/var/run/pkcsslotd.pid
+PIDFILE=/run/opencryptoki/pkcsslotd.pid
```

Comment:
That is a tough one.  From how I read the FHS it seems that the pid-File is supposed to stay directly under `/run`, but that is not really clear.  Personally I think it is better if we add a folder to consistently use this folder for all files.  But I am really unsure here...

Reply:
OK, what would you suggest? 

Stay on the save side and keep the pid file in /run, but put the 2 sockets into the subdirectory?
Or stay et the VERY save side and keep everything in /run ?

Reply:
So lets keep it simple and low risk and keep everything in /run. 
I liked the idea of the subdirectory, but if it is too much trouble, then we maybe better to not use the subdirectory.